### PR TITLE
Fix novel styling - MUI styles overriden by tailwind

### LINKED ISSUE #219: MUI styles are getting overrided with Editor styles
In Next.js, I am dynamically rendering a novel editor. After the rendering is complete, the MUI styles are getting effected, and they are not working as expected. Has anyone else experienced this issue?

### PR DESCRIPTION
**Pull request by acvalencia**
Fix #219

### CODE REVIEW

**Review by pcriadoperez — APPROVED** [review]
LGTM

**Review by jotanarciso — APPROVED** [review]
LGTM

**Comment by srizvi** [comment]
On Line 112 of `packages/core/src/ui/editor/bubble-menu/color-selector.tsx` should `novel-bg-wite` be `novel-bg-white`?

File: packages/core/src/ui/editor/bubble-menu/color-selector.tsx [hunk 1]
```diff
@@ -109,7 +109,7 @@ export const ColorSelector: FC<ColorSelectorProps> = ({
     <Popover.Root open={isOpen}>
       <div className="novel-relative novel-h-full">
         <Popover.Trigger
-          className="novel-flex novel-h-full novel-items-center novel-gap-1 novel-p-2 novel-text-sm novel-font-medium novel-text-stone-600 hover:novel-bg-stone-100 active:novel-bg-stone-200"
+          className="novel-flex novel-h-full novel-items-center novel-gap-1 novel-p-2 novel-text-sm novel-font-medium novel-text-stone-600 novel-bg-wite novel-border-transparent hover:novel-bg-stone-100 active:novel-bg-stone-200"
```
Comment (acvalencia) [inline packages/core/src/ui/editor/bubble-menu/color-selector.tsx:112]: novel-bg-white

**Comment by TsarPrince** [comment]
@steven-tey @pcriadoperez @jotanarciso  if all looks good, can this PR be merged pls? I am using Novel in one of our webapp and we've to push it to production. Thanks for the great product btw!

**Comment by andrewdoro** [comment]
https://novel.sh/docs/quickstart#anatomy Novel exports now unstyled components
[0.2.0-canary.0](https://www.npmjs.com/package/novel/v/0.2.0-canary.0) I'll do a new release soon
I also added a guide for tailiwnd with shadcn-ui
https://novel.sh/docs/guides/tailwind/setup

**Pull request closed without merging**